pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileConfig.java

```diff
@@ -45,6 +49,7 @@ public String toString() {
                 "stateDatabaseFileName='" + stateDatabaseFileName + '\'' +
```

Comment:
It should be `FileConfig` instead of `LogFileSequenceConfig`

Reply:
done